src/rev_ai/apiclient.py

```diff
@@ -120,23 +133,27 @@ def submit_job_local_file(
         payload = {}
         if skip_diarization:
             payload['skip_diarization'] = skip_diarization
+        if skip_punctuation:
+            payload['skip_punctuation'] = skip_punctuation
```

Comment:
Can this logic be extracted into a payload_creator and used by both submit job methods?